docs/reference/README.md

```diff
@@ -4,3 +4,15 @@
 - [Build API](build.md)
 - [Eventing API](eventing/eventing.md)
 - [Event Sources API](eventing/eventing-sources.md)
+
+Note: The Knative API reference documentation is manually generated using the
```

Comment:
I'd probably start a new section here like

    ## Updating API Reference docs (for Knative maintainers)